settings/src/components/webauthn/list-keys.js

```diff
@@ -1,38 +1,160 @@
 /**
  * WordPress dependencies
  */
-import { Button } from '@wordpress/components';
+import { Button, Modal, Notice, Spinner } from '@wordpress/components';
+import { useCallback, useContext, useState } from '@wordpress/element';
+import { Icon, cancelCircleFilled } from '@wordpress/icons';
 
-const confirm = window.confirm;
+/**
+ * Internal dependencies
+ */
+import { GlobalContext } from '../../script';
+import { refreshRecord } from '../../utilities';
+
+/**
+ * Global dependencies
+ */
+const ajax = wp.ajax;
```

Comment:
Why not use `apiFetch` the same as the other components:

https://github.com/WordPress/wporg-two-factor/blob/5fe83cf9453625836e4e82dfe8a608835ef901be/settings/src/components/totp.js#L50

Reply:
This particular endpoint is admin-ajax, rather than the REST API. It's from the `two-factor-provider-webauthn` plugin, rather than one of our custom endpoints.

IIRC, using `apiFetch` would require a lot of customization since it's only intended to be used with REST endpoints. It'd probably be easier to use native `fetch` if we wanted a modern JS API.

`wp.ajax` felt like like the best approach, though, since it's built specifically for admin-ajax endpoints.